homeassistant/components/homekit_controller.py

```diff
@@ -0,0 +1,178 @@
+"""
+Support for Homekit device discovery.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/homekit_controller/
+"""
+import http
+import json
+import logging
+import os
+import uuid
+
+from homeassistant.components.discovery import SERVICE_HOMEKIT
+from homeassistant.config import get_default_config_dir
+from homeassistant.helpers import discovery
+from homeassistant.helpers.entity import Entity
+
+REQUIREMENTS = ['homekit==0.5']
+
+homekit_dir = 'homekit'
```

Comment:
Since we don't want users to edit files in this dir, let's prefix the name with a `.`

Reply:
Done